packages/tests/tests/store.node.spec.ts

```diff
@@ -314,6 +322,9 @@ describe("Waku Store", () => {
     const asymEncoder = new AsymEncoder(asymTopic, publicKey);
     const symEncoder = new SymEncoder(symTopic, symKey);
 
+    const storeEncoder = new EncoderV0(TestContentTopic, false);
```

Comment:
Create a new test (`it` block) for ephemeral instead of adding on an existing and already long/complex test.

The new test can be `skip` for now (`it.skip`) due to https://github.com/status-im/nwaku/issues/1371